### PR TITLE
Financial feature specs

### PR DESCRIPTION
**Pull request by pjsumm**
#584 
#585
#1436
#1437 
#1438 
#1439 
#1440

### CODE REVIEW

File: specs/features/step_definitions/financial.js [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+    .then(() => { return setText('.namedebt .first input', 'Bruce') })
+    .then(() => { return setText('.namedebt .middle input', 'F') })
+    .then(() => { return setText('.namedebt .last input', 'Wayne') })
+    .then(() => { return setText('input[name="CourtInvolved"]', 'Hall of Justice') })
+    .then(() => { return setText('.address .fields .mailing input', '1234 Some Rd') })
+    .then(() => { return setText('.address .fields .city input', 'Metropolis') })
+    .then(() => { return setText('.address .fields .state input', 'NY') })
+    .then(() => { return setText('.address .fields .zipcode input', '85010') })
+    .then(() => { return tab('.address .fields .zipcode input') })
+    .then(() => { return client.pause(3000) })
```
Comment (bmallred) [inline specs/features/step_definitions/financial.js]: Nice! There is a helper to do addresses as well and a valid address in one of the tests if you wanted to bypass the suggestion dialog.

File: specs/features/step_definitions/financial.js [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+    .then(() => { return setText('.content .card-description textarea', 'This is a test description') })
+}
+
+const completeCredit = (promise) => {
+  return promise
+    .then(() => { return setOption('.credit-counseling .credit-branch .yes') })
+    .then(() => { return setText('.credit-counseling .credit-explanation textarea', 'This is a test explanation') })
+    .then(() => { return setText('input[name="Name"]', 'Credit Counseling Organization Name') })
+    .then(() => { return setDomesticTelephone('.credit-counseling .telephone', '703', '111', '2222', 'Cell') })
+    .then(() => { return setDomesticAddress('.credit-counseling .credit-address', '13709 Walsingham Rd', 'Largo', 'FL', '33774') })
```
Comment (bmallred) [inline specs/features/step_definitions/financial.js]: Never mind my previous comment... looks like you knew about the helper function

File: specs/features/step_definitions/financial.js [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+    .then(() => { return setText('.delinquent .delinquent-date .year input', '2001') })
+    .then(() => { return setText('input[name="CourtName"]', 'Sample Court Name') })
+    .then(() => { return setDomesticAddress('.delinquent .delinquent-courtaddress', '13709 Walsingham Rd', 'Largo', 'FL', '33774') })
+    .then(() => { return setText('.delinquent .delinquent-description textarea', 'This is a test description') })
+}
+
+const completeNonpayment = (promise) => {
+  return promise
+    .then(() => { return setOption('.nonpayment .nonpayment-branch .yes') })
+    .then(() => { return setText('input[name="Name"]', 'Organization debt is or was owed') })
```
Comment (floresj) [inline specs/features/step_definitions/financial.js]: Where's my money.

Comment (bmallred) [inline specs/features/step_definitions/financial.js]: https://www.youtube.com/watch?v=LdF_Vo4B6Ms

Comment (billy22g) [inline specs/features/step_definitions/financial.js]: lol

**Pull request merged**